spec/factory_girl/attachments.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :attachment do
+    file "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.